webhooks_skeleton.js

```diff
@@ -0,0 +1,10 @@
+const sqlite3 = require('sqlite3')
+
+let db = new sqlite3.Database('webhooks.db');
+let sqlQuery = 'SELECT * FROM transactions';
+
+db.all(sqlQuery, [], (err, rows) => {
```

Comment:
This will load everything, which I think is perfectly okay to get candidates started! We can ask them about setting up paging if there's extra time during an interview.